.gitignore

```diff
@@ -1,6 +1,11 @@
 # Ape stuff
 .build/
 
+# NPM Stuff
+node_modules/
+package.json
```

Comment:
does `package.json` get generated? It isn't a typical file to exclude, that is why I am asking